cuda_bindings/setup.py

```diff
@@ -238,6 +239,8 @@ def generate_output(infile, local):
     os.path.dirname(sysconfig.get_path("include")),
 ] + include_path_list
 library_dirs = [sysconfig.get_path("platlib"), os.path.join(os.sys.prefix, "lib")]
+cudalib_subdir = r"lib\x64" if sys.platform == "win32" else "lib64"
```

Comment:
On the Linux side, one annoying thing is that depending on how CUDA is installed, this could be `lib` or `lib64`:
```shell
$ ls -l /usr/local/cuda-13.0/lib64/libcudart_static.a 
-rw-r--r-- 1 root root 1361104 Aug  5 09:32 /usr/local/cuda-13.0/lib64/libcudart_static.a
$ ls -l /usr/local/cuda-13.0/targets/x86_64-linux/lib/libcudart_static.a 
-rw-r--r-- 1 root root 1361104 Aug  5 09:32 /usr/local/cuda-13.0/targets/x86_64-linux/lib/libcudart_static.a
``` 

https://github.com/conda-forge/cuda-cudart-feedstock/blob/38fb08899691fa53fce33960ef37d48ab58584b0/recipe/meta.yaml#L220

https://github.com/conda-forge/cuda-cudart-feedstock/blob/38fb08899691fa53fce33960ef37d48ab58584b0/recipe/meta.yaml#L251

We probably should support both.